File: app/src/main/java/com/vladislav/workoutassistant/data/db/dao/SetDao.java
package com.vladislav.workoutassistant.data.db.dao;

import com.vladislav.workoutassistant.data.db.entity.Set;

import java.util.List;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.OnConflictStrategy;

@Dao
public interface SetDao {

    @Insert(onConflict = OnConflictStrategy.REPLACE)
    void insertSets(List<Set> sets);
}

File: app/src/main/java/com/vladislav/workoutassistant/data/db/dao/SetDao.kt
package com.vladislav.workoutassistant.data.db.dao

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.OnConflictStrategy
import com.vladislav.workoutassistant.data.db.entity.Set

@Dao
interface SetDao {

    @Insert(onConflict = OnConflictStrategy.REPLACE)
    fun insertSets(sets: List<Set>)
}
